person/views.py

```diff
@@ -48,6 +50,26 @@ def godfather_list(request):
 
     form = FilterGodfatherForm(request.GET or None)
     objects = godfather_filter(objects, form)
+    if request.method == "POST":
+        try:
+            queryset = objects
+            data = queryset.values()
+            df = pd.DataFrame.from_records(data)
+            excel_file = io.BytesIO()
+            field_names = df.columns
+            verbose_names = [GodFather._meta.get_field(field_name).verbose_name for field_name in field_names]
+            # Map the field names to the verbose names in the DataFrame columns
+            df.columns = verbose_names
+            df.to_excel(excel_file, index=False)
+            excel_file.seek(0)
+            response = HttpResponse(excel_file.read(), content_type='application/vnd.ms-excel')
+            response['Content-Disposition'] = 'attachment; filename=padrinos.xlsx'
+            excel_file.close()
+            return response
```

Comment:
It is missing to add the comment of correct operation.

Reply:
I delete the correct messages because they were displayed in a wrong way, and I don't think they are necessary